frontend/app/framework/angular/pipes/date-time.pipes.spec.ts

```diff
@@ -98,6 +99,20 @@ describe('FromNowPipe', () => {
         expect(actual).toBe(expected);
     });
 
+    it('should format to from now string in Dutch', () => {
+        try {
+            DateHelper.setlocale('nl');
```

Comment:
I know I am picky, but can you move this call, before the try? Then everything inside the try is the actual test and the stuff outside is the preparation.

Reply:
Its better to be picky than to be sloppy. I move it.